tests/integration/config.yml

```diff
@@ -0,0 +1,128 @@
+eden:
+    #root directory of eden
+    root: .
+    images:
+        #directory to save images
+        dist: /tmp
+
+        #yml to build docker image
+        docker: /dev/null
+
+        #yml to build vm image
+        vm: /dev/null
+
+    #download eve instead of build
+    download: true
+
+    #eserver is tool for serve images
+    eserver:
+        #port for eserver
+        port: 8888
+
+        #pid for eserver
+        pid: eserver.pid
+
+        #log of eserver
+        log: eserver.log
+
+    #directory to save certs
+    certs-dist: certs
+
+    #directory to save binaries
+    bin-dist: bin
+
+    #ssh-key to put into EVE
+    ssh-key: ../../conf/ssh.key
+
+    #observe logs in tests
+    logs: false
+
+#config is generated by eden
+adam:
+    #location of adam
+    dist: ../../dist/amd64/installer 
+
+    #port of adam
+    port: 6000
+
+    #domain of adam
+    domain: localhost
+
+    #ip of adam for EVE access
+    eve-ip: 127.0.0.1
+
+    #ip of adam for EDEN access
+    ip: 127.0.0.1
+
+    #force adam rebuild
+    force: false
+
+    #certificate for communication with adam
+    ca: ""
+
+eve:
+    #EVE arch (amd64/arm64)
+    arch: amd64
+
+    #EVE os (linux/darwin)
+    os: darwin
```

Comment:
Why does it matter? You're running EVE under qemu, right?

Reply:
That's part of the development happening in Eden (and hence config having more items than you care about) https://github.com/lf-edge/eden -- in fact these tests are actually imported from Eden: https://github.com/lf-edge/eden/tree/master/tests/integration

They may go away soon['ish] once we figure out the ultimate integration between Eden and EVE. 

The point of keeping it in EVE repo for now is to try out the workflow (as suggested by Avi) where integration tests are developed as part of EVE repo. So consider this a trial period.

That said -- given the recent improvements in Eden -- some of these config items can be removed (including the ones you pointed at) -- so let me minimize the config before I push.